.github/workflows/sanity.yml

```diff
@@ -78,8 +78,9 @@ jobs:
       - name: Policy Describe
         run: go run ./main.go policy describe k8s//nsa_cisa_v1/pod_security --config=internal/test/test_config.hcl
 
-      - name: Policy Download
-        run: go run ./main.go policy download github.com/roneli/aws//cis_v1.2.0 --config=internal/test/test_config.hcl
+      # TODO: Fix this in later PR
```

Comment:
We might be able to use the approach in https://github.com/cloudquery/cloudquery/pull/848/files#diff-3fb3b260bf8d8453a9e3eed595af8d91e1f28d5c5681743103007b1d60c96a51R48 as an alternative

Reply:
Yeah potentially - it's another issue but I don't think this should be defined in the GH workflow. If we use this appraoch it should be defined under `scripts/sanity.sh` and then also added to make file so both the GH action  and the developer can run this in the same way.

Reply:
Added this in this PR